src/multi_vector_simulator/utils/__init__.py

```diff
@@ -158,10 +175,28 @@ def compare_input_parameters_with_reference(folder_path, ext=JSON_EXT):
 
             for sp in not_matching_params:
                 if sp in required_parameters[mp]:
-                    # the sub parameter is not provided but is required --> missing
-                    param_list = missing_parameters.get(mp, [])
-                    param_list.append(sp)
-                    missing_parameters[mp] = param_list
+
+                    if sp in KNOWN_EXTRA_PARAMETERS and set_default is True:
+                        # the sub parameter is not provided but is listed in known extra parameters
+                        # --> default value is set for this parameter
+                        main_parameters[mp][sp] = {
+                            UNIT: KNOWN_EXTRA_PARAMETERS[sp][UNIT],
+                            VALUE: KNOWN_EXTRA_PARAMETERS[sp][DEFAULT_VALUE],
+                        }
```

Comment:
Reminder: To fix #755 we will need to change this.

Reply:
Actually, I think #755 is fixed by this PR ...